internal/app/go-filecoin/paymentchannel/mgr_state_view_api.go

```diff
@@ -0,0 +1,64 @@
+package paymentchannel
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-fil-markets/shared"
+	"github.com/ipfs/go-cid"
+
+	"github.com/filecoin-project/go-filecoin/internal/pkg/block"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/cborutil"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/encoding"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/state"
+)
+
+// ManagerStateViewer is a wrapper for state viewer and state view to fulfill requirements for
+// the paymentchannel.Manager
+type ManagerStateViewer struct {
```

Comment:
Extracting this struct and functionality simplified unit and integration testing.